go-controller/pkg/node/gateway_localnet_linux_test.go

```diff
@@ -41,7 +41,7 @@ func initFakeNodePortWatcher(fakeOvnNode *FakeOVNNode, iptV4, iptV6 util.IPTable
 
 	fNPW := localPortWatcherData{
 		recorder:     fakeOvnNode.recorder,
-		gatewayIP:    v4localnetGatewayIP,
+		gatewayIPs:   map[bool]string{false: v4localnetGatewayIP},
```

Comment:
There is a compilation  error here as gatewayIPs is no longer available as it has been separated into v4 and v6.